### PR TITLE
feat(dashboard): connect create api

### PR DESCRIPTION
**Pull request by minseolee**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description

It works~~!

### CODE REVIEW

**Review by piggggggggy — APPROVED** [review]
LGTM!!

File: src/services/dashboards/dashboard-create/DashboardCreatePage.vue [hunk 1]
```diff
@@ -87,25 +87,27 @@ export default {
...
             } else {
                 _dashboardTemplate = {
                     ...dashboardTemplate.value,
+                    dashboard_id: undefined,
                     viewers: state.dashboardViewerType,
                 };
             }
 
-            dashboardDetailStore.setDashboardInfo(_dashboardTemplate);
+            dashboardDetailStore.setDashboardInfo(_dashboardTemplate, undefined);
```
Comment (yuda110) [inline src/services/dashboards/dashboard-create/DashboardCreatePage.vue:110]: suggestion: How about this?

```suggestion
            dashboardDetailStore.setDashboardInfo(_dashboardTemplate);
            dashboardDetailStore.dashboardId = undefined;
```

**Review by yuda110 — APPROVED** [review]
LGTM!!!

File: src/services/dashboards/dashboard-create/DashboardCreatePage.vue (hunk 1 above)
Comment (minseolee) [reply src/services/dashboards/dashboard-create/DashboardCreatePage.vue:110]: Oh it looks good~!

**Pull request merged**